README.md

```diff
@@ -43,6 +48,28 @@ The module currently does not support setting the following FSCloud requirements
 
 Tracking issue with IBM provider -> https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4204
 
+### Pre-wired CBR configuration for FS Cloud
+
+This module creates default coarse-grained CBR rules in a given account following a "secure by default" approach - that is: deny all flows by default, except known documented communication in the [Financial Services Cloud Reference Architecture](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-vpc-architecture-about):
+
+- Cloud Object Storage (COS) -> Hyper Protect Crypto Services (HPCS)
+- Block Storage -> Hyper Protect Crypto Services (HPCS)
+- IBM Cloud Kubernetes Service (IKS) -> Hyper Protect Crypto Services (HPCS)
+- All IBM Cloud Databases (ICD) services -> Hyper Protect Crypto Services (HPCS)
+- Activity Tracker route -> Cloud Object Storage (COS)
+- Virtual Private Clouds (VPCs) where clusters are deployed -> Cloud Object Storage (COS)
+- IBM Cloud VPC Infrastructure Services (IS) -> Cloud Object Storage (COS)
+- Virtual Private Cloud workload (eg: Kubernetes worker nodes) -> IBM Cloud Container Registry
+- IBM Cloud Databases (ICD) -> Hyper Protect Crypto Services (HPCS)
+- IBM Cloud Kubernetes Service (IKS) -> VPC Infrastructure Services (IS)
+- Event Streams (Messagehub) -> Hyper Protect Crypto Services (HPCS)
+
+**Note on KMS**: the module supports setting up rules for Key Protect, and Hyper Protect Crypto Services. By default the modules set rules for Hyper Protect Crypto Services, but this can be modified to use Key Protect, Hyper Protect, or both Key Protect and Hyper Protect Crypto Services using the input variable `kms_service_targeted_by_prewired_rules`.
```

Comment:
the variable is called `cbr_kms_service_targeted_by_prewired_rules` for this module

Reply:
removed this section in favor of linking to the CBR readme